client/app/components/app-view/index.js

```diff
@@ -0,0 +1,68 @@
+import debug from 'debug';
+import template from './template.html';
+
+const logger = debug('redash:app-view');
+
+export default function init(ngModule) {
+  let customHandler = null;
```

Comment:
How about having an object to handle errors:

```javascript

const errorHandler = {
    error: null,
    reset() {
        this.error = null;
    },
    processError(error) {
        this.error = error;
    }
}
```

Then we can use it without waiting for it to be initialized.

i.e.:

```javascript
ngModule.factory('$exceptionHandler', () => errorHandler.processError);
```